File: source/fizzbuzz.cpp
#include<iostream>

int main()
{
     for(int foo = 0; foo <= 100; ++foo)
     {
          if(foo % 3 == 0)
          {
               if(foo % 5 == 0)
               {
                    std::cout << "fizzbuzz" << std::endl;
               }
               else
               {
                    std::cout << "fizz" << std::endl;
               }
          }
          else if(foo % 5 == 0)
          {
               if(foo % 3 == 0)
               {
                    std::cout << "fizzbuzz" << std::endl;
               }
               else
               {
                    std::cout << "buzz" << std::endl;
               }
          }
          else
          {
               std::cout << foo << std::endl;
          }
     }
     return 0;
}

File: source/fizzbuzz.py
counter = 0
while counter <= 100:
    counter = counter + 1
    if counter % 3 == 0:
        print("fizz")
    elif counter % 5 == 0:
        print("buzz")
    elif counter % 3 == 0 and counter % 5 == 0:
        print("fizzbuzz")
    else:
        print(counter)
